ci/CMakeLists.txt

```diff
@@ -16,18 +16,30 @@
 cmake_minimum_required(VERSION 2.8)
 project(ovms)
 set(CMAKE_CXX_STANDARD 17)
+message("CMake version:" ${CMAKE_VERSION} )
 
+message("Project source directory: " ${PROJECT_SOURCE_DIR})
 file(GLOB ovms_SRC
-    "*.hpp"
-    "*.cpp"
+    "${PROJECT_SOURCE_DIR}/*.hpp"
+    "${PROJECT_SOURCE_DIR}/*.cpp"
 )
 
+message("OVMS sources:" ${ovms_SRC} )
+
 file(GLOB OCR_SRC
-    "custom_node_interface.hpp"
-    "custom_nodes/east_ocr/*.hpp"
-    "custom_nodes/east_ocr/*.cpp"
+    "${PROJECT_SOURCE_DIR}/custom_node_interface.h"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/*/*.hpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/*/*.cpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/east_ocr/utils.hpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/east_ocr/east_ocr.cpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/image_transformation/utils.hpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/image_transformation/image_transformation.cpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/model_zoo_intel_object_detection/model_zoo_intel_object_detection.cpp"
+    "${PROJECT_SOURCE_DIR}/custom_nodes/model_zoo_intel_object_detection/utils.hpp"
 )
 
+message("Custom nodes sources:" ${OCR_SRC} )
+
 add_library(ocr ${OCR_SRC})
```

Comment:
You are adding sources of three libs, yet only one library is added here. How does it work?

Wouldn't it make more sense to have 3 libraries declared separately?